docs/data/data-grid/list-view/list-view.md

```diff
@@ -16,14 +16,99 @@ Unlike the default grid view, the list view makes no assumptions on how data is
 
 In order to display data in a list view, a `unstable_listColumn` prop must be provided with a `renderCell` function.
 
-{{"demo": "ListView.js", "bg": "inline"}}
+```tsx
+function ListViewCell(params: GridRenderCellParams) {
+  return <>{params.row.id}</>;
+}
+
+const listColDef: GridListColDef = {
+  field: 'listColumn',
+  renderCell: ListViewCell,
+};
+```
+
+{{"demo": "ListView.js", "bg": true}}
+
+## Enable with a media query
+
+Use the `useMediaQuery` hook from `@mui/material` to enable the list view feature at a specified breakpoint.
+
+The demo below automatically switches to a list layout when the viewport width is below the `md` breakpoint.
+
+{{"demo": "ListViewMediaQuery.js", "bg": "inline"}}
+
+## Editable rows
+
+The [editing feature](/x/react-data-grid/editing/) is not supported in list view, but it is possible to build an editing experience from within your custom cell renderer, as shown below.
+
+{{"demo": "ListViewEdit.js", "bg": true}}
+
+## Field visibility
+
+Field visibility can be toggled in list view, similarly to the default grid view.
```

Comment:
I'm wondering if "Advanced usage" section also covers this aspect and if we could skip this section (for simplicity) 🤔 
What do you think?

Reply:
Yeah, in hindsight I don't think it is necessary to have a section for this.